iocage_lib/ioc_json.py

```diff
@@ -2452,6 +2458,18 @@ def json_check_prop(self, key, value, conf):
                     IOCRCTL.validate_rctl_props(key, value)
                 elif key == 'cpuset':
                     IOCCpuset.validate_cpuset_prop(value)
+                elif key == 'localhost_ip':
```

Comment:
I think we need to account for `none` here ? Else we will raise an error on that one too

Reply:
None is an error in this case, since it shouldn't be specified otherwise